src/clj/witan/send/report.clj

```diff
@@ -0,0 +1,23 @@
+(ns witan.send.report
+  (:require [clojure.java.io :as io]
+            [clj-time.core :as t]
+            [clj-time.format :as f]
+            [clojure.java.shell :as sh]))
+
+(def send-report (atom []))
+
+(defn reset-send-report []
+  (sh/sh "git" "fetch" "--tags")
+  (reset! send-report [(str "SEND Model " (:out (sh/sh "git" "describe" "--abbrev=0" "--tags")))])
```

Comment:
If you're going to use `reset!` here, I'd take all the subsequent strings and add them into the vector so you're doing a single transformation on that atom.